packages/financial-templates-lib/src/logger/SlackTransport.ts

```diff
@@ -195,8 +197,8 @@ class SlackHook extends Transport {
         if (response.status != 200) errorThrown = true;
       }
     }
+    if (errorThrown) return callback(new TransportError("Slack", new Error("Axios response error"), info));
```

Comment:
This still does not cover network errors. Will create a follow-up PR to enclose the request in try/catch, similarly as in other transports